src/Gui/Document.py

```diff
@@ -0,0 +1,257 @@
+from ..Base.Metadata import metadata
```

Comment:
I think we should avoid relative imports. Somewhere the absolute path should be set, and then in all files, all import will always be relative to `src/`